buildscripts/packaging/Windows/PortableApps/appinfo.ini.in

```diff
@@ -0,0 +1,33 @@
+[Format]
+Type=PortableApps.comFormat
+Version=3.7 # Version of the PortableApps format, not of MuseScore
+
+[Details]
+Name=Audacity Portable
+AppId=AudacityPortable
+Publisher=Audacity BVBA and Others
```

Comment:
The publisher should be updated too. BVBA is a Belgian abbreviation for LTD/LLC.